src/include/sof/audio/module_adapter/library/loadable_processing_module.h

```diff
@@ -8,7 +8,8 @@
 #ifndef _LOADABLE_PROCESSING_MODULE_H_
 #define _LOADABLE_PROCESSING_MODULE_H_
 
-#include  <sof/audio/module_adapter/iadk/adsp_stddef.h>
+#include <sof/audio/module_adapter/iadk/adsp_stddef.h>
+#include <sof/audio/module_adapter/library/module_api_ver.h>
```

Comment:
this doesn't look right. You haven't changed anything in the header, so there's no reason why it now need that additional header